app/Http/Controllers/EducatorResourcesController.php

```diff
@@ -53,28 +52,30 @@ public function index(Request $request)
           });
 
         $audienceOptions = ResourceCategory::where('type', 'audience')
-            ->get()
-            ->map(function ($category) {
-                return [
-                  'label' => $category->name,
-                  'active' => null,
-                  'ajaxScrollTarget' => null,
-                  'id' => null,
-                  'data-button-value' => Str::kebab(Str::lower($category->name))
-                ];
-            });
+          ->orderBy('position')
+          ->get()
+          ->map(function ($category) {
+              return [
+                'label' => $category->name,
+                'active' => null,
+                'ajaxScrollTarget' => null,
+                'id' => null,
+                'data-button-value' => Str::kebab(Str::lower($category->name))
```

Comment:
I think this is equivalent to `Str::slug()`: https://laravel.com/docs/11.x/strings#method-str-slug